src/controllers/CheckYourAppealController.ts

```diff
@@ -11,11 +11,14 @@ import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController
 import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
 import { Appeal, APPEALS_KEY } from 'app/models/Appeal'
 import { EmailService } from 'app/modules/email-publisher/EmailService'
+import { AppealSubmissionService } from 'app/service/AppealSubmissionService';
 import { CHECK_YOUR_APPEAL_PAGE_URI, CONFIRMATION_PAGE_URI } from 'app/utils/Paths';
 
 @controller(CHECK_YOUR_APPEAL_PAGE_URI, SessionMiddleware, AuthMiddleware)
 export class CheckYourAppealController extends BaseAsyncHttpController {
-    constructor (@inject(EmailService) private readonly emailService: EmailService) {
+
+    constructor (@inject(EmailService) private readonly emailService: EmailService,
+                 @inject(AppealSubmissionService) private readonly appealSubmissionService: AppealSubmissionService) {
```

Comment:
PR does not show conflicts but you will have to merge it with my refactoring.

Reply:
i have merged your changes in and done some refactoring of my code